data.sql

```diff
@@ -16,3 +16,34 @@ INSERT INTO animals
 (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
 ('Devimon', '2015-05-12', 5, true, 11);
 
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Charmander', '2020-02-08', 0, false, 11);
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Plantmon', '2021-11-15', 2, false, 5.7);
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Squirtle', '1993-04-2', 3, false, 12.13);
```

Comment:
- Good job done so far, but the weights for `Charmander`, `Plantmon`, `Squirtle`, and `Angemon`, please fix this so you could have the right query result 👍 👍